stimela/tests/acceptance_tests/stimela-test-kat7.py

```diff
@@ -0,0 +1,461 @@
+                        "nan"           : True,
+                        "flag"          : "legacy",
+                },
+                        input=INPUT, output=OUTPUT,
+                        label="backup_initial_flags:: Backup selfcal flags")
+
+                #First selfcal round
+                recipe.add("cab/calibrator", "calibrator_Gjones_subtract_lsm0", {
+                        "skymodel"           : "%s.lsm.html:output"%(lsm0),
+                        "msname"             : MS,
+                        "threads"            : 16,
+                        "column"             : "DATA",
+                        "output-data"        : "CORR_RES",
+                        "Gjones"             : True,
+                        "Gjones-solution-intervals" : [20,0],     #Ad-hoc right now, subject to change
+                        "Gjones-matrix-type" : "GainDiagPhase",
+                        "tile-size"          : 512,
+                        "field-id"           : 0,
+                },
+                        input=INPUT, output=OUTPUT,
+                        label="calibrator_Gjones_subtract_lsm0:: Calibrate and subtract LSM0")
+
+                # Diversity is a good thing... lets add some DDFacet to this soup bowl
+                imname=PREFIX+'ddfacet'
+                recipe.add("cab/ddfacet", "ddfacet_test",
+                        {
+                                "Data-MS": [MS],
+                                "Output-Name": imname,
+                                "Image-NPix": 256,
+                                "Image-Cell": 30,
+                                "Cache-Reset": True,
+                                "Freq-NBand": 2,
+                                "Weight-ColName": "WEIGHT",
+                                "Beam-Model": "FITS",
+                                "Beam-FITSFile": "'JVLA-L-centred-$(xy)_$(reim).fits'",
+                                "Data-ChunkHours": 10,
+                                "Beam-FITSFeed" : "rl",
+                                "Data-Sort": True,
+                                "Log-Boring": True,
+                                "Deconv-MaxMajorIter": 1,
+                                "Deconv-MaxMinorIter": 500,
+                        },
+                        input=INPUT, output=OUTPUT, shared_memory="36gb",
+                        label="image_target_field_r0ddfacet:: Make a test image using ddfacet")
+
+
+                #Stitch LSMs together
+                lsm2=PREFIX+'-LSM2'
+#               recipe.add("cab/tigger_convert", "stitch_lsms1", {
+#                       "input-skymodel"  :   "%s.lsm.html:output" % lsm0,
+#                       "output-skymodel" :   "%s.lsm.html:output" % lsm1,
+#                       "rename"	  : True,
+#                       "force"           : True,
+#               },
+#                       input=INPUT, output=OUTPUT,
+#                       label="stitch_lsms1::Create master lsm file")
```

Comment:
@SpheMakh what happened to the second round of amplitude calibration? It was there to test that the modes are working correctly

Reply:
I don't think this is a test stimela should make. It's up to the contained package to test that level of detail, we just want to ensure that cab can run.